db/view/view.cc

```diff
@@ -1830,10 +1835,9 @@ future<> view_builder::start(service::migration_manager& mm) {
             // Waited on indirectly in stop().
             (void)_build_step.trigger();
             return make_ready_future<>();
+        }).handle_exception([&aborted] (std::exception_ptr eptr) {
+            vlogger.log(aborted ? log_level::debug : log_level::error, "start failed: {}", eptr);
```

Comment:
Instead of all this complexity of storing a boolean "aborted" earlier and checking it here, wouldn't it be much easier to just have a first handle_exception here, which handles **only** seastar::sleep_aborted and ignores it? It doesn't even matter where the "sleep_aborted" came from (maybe one of the other functions has a sleep too?) - if a sleep was aborted causing the whole process to be aborted, this will be a debug-log.

You can even print a different message in the abort case - e.g., "start aborted" instead of "start failed: sleep_aborted").

Reply:
> Instead of all this complexity of storing a boolean "aborted" earlier and checking it here, wouldn't it be much easier to just have a first handle_exception here, which handles **only** seastar::sleep_aborted and ignores it? It doesn't even matter where the "sleep_aborted" came from (maybe one of the other functions has a sleep too?) - if a sleep was aborted causing the whole process to be aborted, this will be a debug-log.
> 
> You can even print a different message in the abort case - e.g., "start aborted" instead of "start failed: sleep_aborted").

that's possible